app/models/effective/tagging.rb

```diff
@@ -0,0 +1,41 @@
+module Effective
+  class Tagging < ApplicationRecord
+    self.table_name = (EffectivePages.taggings_table_name || :taggings).to_s
+
+    belongs_to :tag, class_name: 'Effective::Tag'
+    belongs_to :taggable, polymorphic: true
+
+    validates :tag,      presence: true
+    validates :taggable, presence: true
```

Comment:
We don't need these validations because they're implied by the belongs_to. We'll get double validation errors this way